app/src/organisms/Devices/ProtocolRun/SetupLabware/SetupLabwareList.tsx

```diff
@@ -48,7 +55,7 @@ const LABWARE_CARD_STYLE = css`
 const StyledTable = styled.table`
   width: 100%;
   text-align: ${TYPOGRAPHY.textAlignLeft};
-  table-layout: ${POSITION_FIXED};
+  table-layout: auto;
```

Comment:
we can use a const from spacing.ts

```suggestion
  table-layout: ${SPACING.spacingAuto};
```

Reply:
Maybe we can use props for L65 with L330?

```ts
height: ${props => (props.requiredborder? ${SPACING.spacing2} : "")};
```